NEW_PROJECTS.md

```diff
@@ -0,0 +1,70 @@
+Note that when contributing a project to SPIFFE, its purpose becomes to serve the SPIFFE community rather than any personal gain. Additionally, we expect a sufficient number of maintainers for a project as it grows, not all of whom may bring the same direction you may originally have had in mind.
+Carefully consider this change in control and purpose.
+
+*Every SPIFFE project* at minimum must follow the [SPIFFE Code of Conduct](/CODE-OF-CONDUCT.md).
+
+## Project Proposal
+
+### Contact
+
+The SSC can be contacted with proposal details at ssc@spiffe.io or on [our SPIFFE slack channel](https://spiffe.slack.com/archives/C01GQ267JJU).
+
+### Proposal Outline
+
+The following details are expected for proposals to help us best understand how the project will serve the community.
+
+Additionally, the information here make great initial `README.md`, `CONTRIBUTING.md`, and `GOVERNANCE.md` files for the repo!
+
+#### Initial Maintainer List
+
+The list of initial maintainers of the project, including GitHub handle.
+
+#### Project Purpose and Direction
+
+What need is the project solving for the community? What problem exists or what enhancement will this create?
+
+What is the *direction* of the project? This is similar but distinct from the *purpose* of the project.
+
+While the purpose highlights the "Why" and "What" of the project, the direction should be about the "How".
+This could include:
+- What does "done" mean for the project? What is the idealized vision for the shape of the project? What are the milestones to complete for each level of [maturity](/MATURITY.md)?
+- What are practices or styles that will be encouraged or discouraged?
+- What notable features or options are in-scope or out-of-scope for the project? This especially will help differentiate your project from any similar ones that might already exist.
+
+It's ok if there's not fully fleshed-out answers to each of these, and projects evolve over time. However the more information that can be provided the more easily SSC may understand the proposal, and the more confidence there will be in alignment amongst the maintainers.
+
+#### Governance
+
+The operations of the project.
+
+How will maintainers be added or removed? What is the goal number of maintainers?
+
+How often will the maintainers meet?
+
+How will maintainers interact with contributors and review their Pull Requests and Issues? Slack? Dedicated online meetings? What is the goal turn around time?
+
+#### Similar Projects
+
+Do similar projects already exist actively within the SPIFFE community? If so, what makes this project sufficiently unique to justify any overlap?
+
+Do similar projects exist outside of SPIFFE that can better help us understand the project's value?
+
+Has this project existed in the past but was abandoned or closed out? What is different in today's world?
+
+## Project Review
```

Comment:
Shall we specify a timeframe for the review process?

Reply:
For the review, would we require any documents? pre-meetings?

Reply:
+ 
Offering examples and template propsal(s) can streamline the process and clarify some doubts.

Reply:
Maybe a FAQ section (though, the doc is satisfactory enough in itself)

Reply:
> Shall we specify a timeframe for the review process?

if within SSC we can come up with a consensus soon on a timeframe, sure. I was anticipating for this initial draft that that may not be something we could do right now, especially without real practice at it

> Offering examples and template propsal(s) can streamline the process and clarify some doubts.

We could take as a follow-up PR, or ask some existing projects to retroactively go through this process so we can point to theirs?

Reply:
added "a couple SSC sync cycles" as initial guidance. we really should be able to have a decision within that time period I think for most cases